src/lib/utils/permissions.test.ts

```
@@ -0,0 +1,189 @@
+	});
+
+	describe('requirePermission()', () => {
+		it('should not throw error when user has permission', () => {
+			expect(() => requirePermission(mockUser, PERMISSIONS.CREATE_USER)).not.toThrow();
+		});
+
+		it('should throw 403 error when user does not have permission', () => {
+			try {
+				requirePermission(mockUser, PERMISSIONS.DELETE_USER);
+				expect.fail('Should have thrown an error');
+			} catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)